src/app.js

```diff
@@ -1 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFile(currentLocation, specifiedLocation) {
```

Comment:
maybe it is better to use relative path   `const fileToCopyPath = path.join(__dirname, fileToCopyPath);` from my perspective it would be more obvious